test/e2e-common.sh

```diff
@@ -238,6 +238,6 @@ function test_teardown() {
   echo ">> Removing test resources (test/config/)"
   ko delete --ignore-not-found=true --now -f test/config/
   echo ">> Removing test namespace"
-  kubectl delete all --all --ignore-not-found --now -n serving-tests
-  kubectl delete --ignore-not-found --now namespace serving-tests
+  kubectl delete all --all --ignore-not-found --now --timeout 60s -n serving-tests
+  kubectl delete --ignore-not-found --now --timeout 600s namespace serving-tests
```

Comment:
dumb question, if the first delete should have deleted all the objects in the namespace, what is the reason to wait 10min to delete the namespace

Reply:
The timeouts were based on what I observed, the second delete somehow still takes some time for reason I don't know. Normally it would take less than 1 minute, I'm giving it 10 times more just to be safe

Reply:
i rather have it at 60s, as this seems to be a known issue, https://www.google.com/url?sa=j&url=https%3A%2F%2Fgithub.com%2Fkubernetes%2Fkubernetes%2Fissues%2F60807%23issuecomment-466278490&uct=1553196026&usg=KgMyeH-4IRxoZdsacyyG7WRbSKI.&source=chat

waiting longer isnt really any safer, it is just stuck.
i rather increase later if it is found to be problematic.

Reply:
sg, updated

Reply:
lol , that's the same thread I read a couple minutes ago